8.platemap/scripts/make_platemap.r

```diff
@@ -0,0 +1,61 @@
+library(ggplot2)
+library(platetools)
+
+# set path to data
+data_path <- file.path("..","..","data","platemap_6hr_4ch.csv")
+figure_path <- file.path("..","figures")
+# make sure the figure path exists
+dir.create(figure_path, showWarnings = FALSE)
+# figure file name
+figure_file <- file.path(figure_path, "platemap_6hr_4ch.png")
+# read in data
+data <- read.csv(data_path)
+head(data)
+
+# format the well column to remove "-"
+data$well <- gsub("-", "", data$well)
+# make dose categorical
+data$dose <- as.factor(data$dose)
+# set order of dose levels
+unique(data$dose)
+data$dose <- factor(data$dose, levels = c(
+    "0",
+    "0.61",
+    "1.22",
+    "2.44",
+    "4.88",
+    "9.77",
+    "19.53",
+    "39.06",
+    "78.13",
+    "156.25"
+))
+
+# set colours for each dose
+library(RColorBrewer)
```

Comment:
Consider moving this import to the start of this file instead to keep consistency,